.github/workflows/clp-lint.yaml

```
@@ -43,6 +43,14 @@ jobs:
         name: "Install coreutils (for md5sum)"
         run: "brew install coreutils"
 
-      - name: "Run lint task"
+      - name: "Run lint task: js"
```

```suggestion
      - name: "Lint .js files"
```

Ditto below if you agree.

---

Shall I include `.ts`? Like `Lint .js/.ts files`?